### ISSUE TITLE
chained minecarts

### ISSUE THREAD

**Issue by Goblinmine** [Goblinmine/goblinmines-minecart-tweaks#4]
Connect minecarts with chains

**Comment by vico93** [comment]
If you are going to implement this, i hope you will do the chained minecarts physics work similar to TrainCarts:

https://github.com/Goblinmine/Goblinmines-Minecart-Tweaks/assets/3485505/453a5581-9b6f-4482-ac98-6d4c39befabe

**Comment by Goblinmine** [comment]
If you mean that there is no delay between the first and the last minecart moving, yes, I did not plan to implement a delay.

On the visual side I planed to add a little bit of a space between carts and render a chain that connects them.

**Comment by vico93** [comment]
> If you mean that there is no delay between the first and the last minecart moving, yes, I did not plan to implement a delay.
> 
> On the visual side I planed to add a little bit of a space between carts and render a chain that connects them.

Yeah i meant more to make sure all the carts move in tandem (other mods like Cammie's one sometimes have issues on tight turns where part of the carts want to go back and others go forward, killing any chance for automation).
